mcs/class/corlib/System/BitConverter.cs

```diff
@@ -2,25 +2,23 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System.Diagnostics;
 using System.Runtime.CompilerServices;
-using System.Runtime.InteropServices;
-
-using Internal.Runtime.CompilerServices;
 
 namespace System
 {
-    public static partial class BitConverter
+	public static partial class BitConverter
 	{
 		[Intrinsic]
 		public static readonly bool IsLittleEndian;
 
+#if !MOBILE // Should be PLATFROM_LINUX instead
 		static BitConverter () {
```

Comment:
Why was this added ? The cctor is needed to compute IsLittleEndian.

Reply:
Yeah, I keep forgetting about the inverted logic